pkg/containerwatcher/v1/container_watcher_private.go

```diff
@@ -104,7 +104,7 @@ func (ch *IGContainerWatcher) getSharedWatchedContainerData(container *container
 	watchedContainer := utils.WatchedContainerData{
 		ContainerID: container.Runtime.ContainerID,
 		ImageID:     container.Runtime.ContainerImageDigest,
-		ImageTag:    container.Runtime.ContainerImageName,
+		// we get ImageTag from the pod spec for consistency between different runtimes
```

Comment:
I don't see you fill it, is it only used for the sbom manager?

Reply:
we fill it in https://github.com/kubescape/node-agent/pull/466/files#diff-81ddbadfb415ccbb9c7af84f11668d1aa5e53c34025bf86d4702f16b4e42f045R275